speaker/voice_profile_phrase_result.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+
+package speaker
+
+import (
+	"strings"
+	"unsafe"
+
+	"github.com/Microsoft/cognitive-services-speech-sdk-go/common"
+)
+
+// #include <stdlib.h>
+// #include <stdint.h>
+// #include <speechapi_c_common.h>
+// #include <speechapi_c_result.h>
+// #include <speechapi_c_recognizer.h>
+//
+import "C"
+
+// VoiceProfilePhraseResult contains activation phrases needed to successfully enroll a voice profile.
+type VoiceProfilePhraseResult struct {
+	handle C.SPXHANDLE
+
+	// Activation phrases for voice profile enrollment 
+	Phrases []string
+
+	// ResultID specifies the result identifier.
+	ResultID string
+
+	// Reason specifies status of speech synthesis result.
+	Reason common.ResultReason
+
+	// ErrorDetails presents error details.
+	ErrorDetails string
+
+	// Collection of additional properties.
+	Properties *common.PropertyCollection
+}
+
+// Close releases the underlying resources
+func (result VoiceProfilePhraseResult) Close() {
+	result.Properties.Close()
+	C.recognizer_result_handle_release(result.handle)
+}
+
+// NewVoiceProfilePhraseResultFromHandle creates a VoiceProfilePhraseResult from a handle (for internal use)
+func NewVoiceProfilePhraseResultFromHandle (handle common.SPXHandle) (*VoiceProfilePhraseResult, error) {
```

Comment:
same

Reply:
used in voice_profile_client.GetActivationPhrasesAsync